ios/ReactTestApp/ContentView.swift

```diff
@@ -132,21 +132,21 @@ final class ContentViewController: UITableViewController {
         #endif
     }
 
-    private func navigate(to component: Component, name: String) {
+    private func navigate(to component: Component) {
         guard let bridge = reactInstance.bridge,
               let navigationController = navigationController else {
             return
         }
 
         let viewController: UIViewController = {
-            if let viewController = RTAViewControllerFromString(name, bridge) {
+            if let viewController = RTAViewControllerFromString(component.appKey, bridge) {
```

Comment:
Shouldn't we fallback to `displayName` here too if we want to keep backwards compat?

Reply:
No, this function tries to retrieve the native view controller given its name.